python/ml4ir/applications/classification/pipeline.py

```diff
@@ -168,6 +170,33 @@ def get_relevance_dataset(
 
         return relevance_dataset
 
+    def finish(self, job_status, job_info):
+        # This will write the _SUCCESS/_FAILURE files
+        super().finish(job_status, job_info)
+
+        # Create model_bundle.zip
+        if self.args.file_handler == FileHandlerKey.SPARK:
+            # Copy tfrecord files in model_files
+            os.makedirs("epmm")
+            shutil.copytree(os.path.join(self.models_dir_local, "final/tfrecord/"), "model_files")
+            shutil.make_archive("./epmm/model_bundle", "zip", "model_files")
+
+            (self.file_io
```

Comment:
while this solution is good for the actual problem at hand (let's call it a v0), I am not comfortable with selecting the vocabularies to put in the bundle. I think it would be preferable to copy all vocabulary files.